pkg/apis/pipeline/v1/task_validation_test.go

```diff
@@ -142,7 +142,7 @@ func TestTaskSpecValidatePropagatedParamsAndWorkspaces(t *testing.T) {
 				Workspaces:   tt.fields.Workspaces,
 				Results:      tt.fields.Results,
 			}
-			ctx := config.EnableAlphaAPIFields(context.Background())
+			ctx := context.Background()
```

Comment:
we need to check if beta api fields is enabled in v1

Reply:
sounds good!